requirements-docs.txt

```diff
@@ -141,7 +146,7 @@ mkdocs-material-extensions==1.3.1
 mkdocs-minify-plugin==0.8.0
     # via hatch.envs.docs
 mkdocs-nav-weight==0.2.0
-    # manually added
+    # via hatch.envs.docs
```

Comment:
Thanks for updating this file, I was too lazy to remember how to use hatch when I  made the changes :D